core_js/popup.js

```diff
@@ -155,6 +189,39 @@ function setSwitchButton(id, varname)
     element.checked = this[varname];
 }
 
+/**
+* Adds (or removes) the site the user is on to the whitelist
+* Whitelisted sites do not get filtered
+* @param {boolean} remove If true remove current site instead of adding
+*/
+function changeWhitelist(removeWl = false) {
+    if (removeWl != true) { // Handle click obj
```

Comment:
Assignment is useless. It is better to pass the correct state in the onclick event